cdap-master/src/main/java/co/cask/cdap/data/tools/ReplicationStatusTool.java

```diff
@@ -242,7 +242,7 @@ private static void checkHBaseReplicationComplete(Map<String, Long> masterTimeMa
       System.out.println("Number of regions on the Master and Slave Clusters do not match.");
     }
 
-    complete = !checkDifferences(masterTimeMap.keySet(), slaveTimeMap.keySet(), "Region");
+    complete = !checkDifferences(masterTimeMap.keySet(), slaveTimeMap.keySet(), "Region", true);
```

Comment:
don't think this should be true either. Suppose you stop cdap, dump the master state, and copy it to slave. During this time, some other table with replication has split its regions. That new region won't be in the master state, but it will be in the slave state.